packages/react-reconciler/src/ReactFiberFlags.js

```diff
@@ -12,29 +12,30 @@ import {enableCreateEventHandleAPI} from 'shared/ReactFeatureFlags';
 export type Flags = number;
 
 // Don't change these values. They're used by React Dev Tools.
-export const NoFlags = /*                      */ 0b0000000000000000000000000000;
-export const PerformedWork = /*                */ 0b0000000000000000000000000001;
-export const Placement = /*                    */ 0b0000000000000000000000000010;
-export const DidCapture = /*                   */ 0b0000000000000000000010000000;
-export const Hydrating = /*                    */ 0b0000000000000001000000000000;
+export const NoFlags = /*                      */ 0b00000000000000000000000000000;
+export const PerformedWork = /*                */ 0b00000000000000000000000000001;
+export const Placement = /*                    */ 0b00000000000000000000000000010;
+export const DidCapture = /*                   */ 0b00000000000000000000010000000;
+export const Hydrating = /*                    */ 0b00000000000000001000000000000;
 
 // You can change the rest (and add more).
-export const Update = /*                       */ 0b0000000000000000000000000100;
-/* Skipped value:                                 0b0000000000000000000000001000; */
+export const Update = /*                       */ 0b00000000000000000000000000100;
+export const Cloned = /*                       */ 0b10000000000000000000000000000;
+/* Skipped value:                                 0b00000000000000000000000001000; */
```

Comment:
Could we just use the skipped value? Can't think of any reason why not. I'm guessing this is the old Deletion flag that was replaced by ChildDeletion and then removed

Reply:
It was introduced in d69b2cf8208848b0f71b5214ddff55a1ff437cc8 reverting some changes to the Flags. Some versioning issues with DevTools that I don't fully understand. Maybe it's okay when the devtools have since auto-updated to a newer version?

cc @hoxyq in case you know if this would be safe?

Reply:
`react-devtools-shared/src/backend/ReactFiberFlags.js`  was removed in https://github.com/facebook/react/pull/26542/files, so we could mirror the whole function implementation, instead of just mirroring flags.

We should update these flags for DevTools accordingly, like here, for example: https://github.com/facebook/react/blame/main/packages/react-devtools-shared/src/backend/renderer.js#L1515.

I think it should be safe to use skipped value here, because DevTools were not depending on this flag at any time, AFAIK. I can help with testing these changes if you end up with using skipped value.